internal/test/builder/bootstrap.go

```diff
@@ -35,13 +35,19 @@ var (
 	// GenericBootstrapConfigTemplateCRD is a generic bootstrap template CRD.
 	GenericBootstrapConfigTemplateCRD = untypedCRD(BootstrapGroupVersion.WithKind(GenericBootstrapConfigTemplateKind))
 
+	// GenericBootstrapObjectKind is the Kind for the GenericBootstrapObject.
+	GenericBootstrapObjectKind = "GenericBootstrapObject"
```

Comment:
We should use GenericBootstrapConfigKind instead

We just use Bootstrap instead of BootstrapConfig in a lot of places somehow. But that's not an intentional difference